systest/multi-tenancy/basic_test.go

```diff
@@ -210,7 +210,7 @@ func (suite *MultitenancyTestSuite) TestPersistentQuery() {
 	require.NoError(t, err)
 
 	// Upgrade
-	suite.Upgrade(dstDB, dgraphtest.BackupRestore)
+	suite.Upgrade(dgraphtest.BackupRestore)
```

Comment:
do we need to pass strategy in the Upgrade function?

Reply:
Going forward we will have more than one strategy which we can specify as an argument here.
What are your thoughts ? Hardcode it as BackupRestore ?

Reply:
Yes, makes sense, I will change it and push the changes.